proxy/src/http/session.rs

```diff
@@ -294,6 +294,12 @@ mod test {
                     "appearance": {
                         "theme": "light",
                     },
+                    "coco": {
+                        "seeds": [
+                            "seed.radicle.xyz",
+                            "194.134.54.13"
```

Comment:
Just a note: IP addresses won't work as seeds because they can't carry the PeerId which is needed for the TLS handshake.

Reply:
Could you expand how the PeerID will be encoded in DNS entries? Also if this is the case that we can't accept IP addresses, we should find a way to encode that so that we never are exposed to the risk of constructing a peer "address" incorrectly.

Reply:
PeerIds are planned to be stored in a CNAME entry.

Reply:
To expand here a bit more, the idea is that for a given DNS entry the client is suppose to do two lookups:

* `CNAME` which gives the `PeerId` as the value
* `A` which gives you the `IP` address

Which will result in `<peer_id>@<ip>` as the true location identification of the peer. Common practice as far as identifiers go that depend on some form of node id. Is there spec/docs somewhere?

Reply:
@xla there's a [ticket here](https://github.com/radicle-dev/radicle-upstream/issues/693) for DNS discussion

Reply:
Since these seeds aren't actual seeds I don't see any issue besides potentially confusing the reader.

Reply:
I can remove the IP address one; it's only there because it was in the design spec.